waspc/src/Wasp/AI/GenerateNewProject/WaspFile.hs

```diff
@@ -89,12 +89,20 @@ fixWaspFile newProjectDetails waspFilePath plan = do
             ${compileErrorsText}
 
             Some common mistakes to look for:
+              - using non-default imports in page components 
```

Comment:
```suggestion
              - Using non-default imports in page components. 
```

Reply:
This makes sense to keep it consistent with the rest of the bullet points.